### PR TITLE
run in win10_x64 vs2019

### PR DESCRIPTION
**Pull request by lookingforfeng**
run in win10_x64 vs2019

### CODE REVIEW

File: src/JSBSim.cpp [hunk 1]
```diff
@@ -308,7 +308,7 @@ int main(int argc, char* argv[])
 
 int real_main(int argc, char* argv[])
 {
-  // *** INITIALIZATIONS *** //
+  // *** 初始化 *** //
```
Comment (bcoconni) [inline src/JSBSim.cpp:311]: Please leave the comments in English

File: src/JSBSim.cpp [hunk 2]
```diff
@@ -336,14 +336,14 @@ int real_main(int argc, char* argv[])
   catalog = false;
   nohighlight = false;
 
-  // *** PARSE OPTIONS PASSED INTO THIS SPECIFIC APPLICATION: JSBSim *** //
+  // *** 解析从命令行传入的参数 *** //
   success = options(argc, argv);
```
Comment (bcoconni) [inline src/JSBSim.cpp:340]: Same as above : leave the comments in English.

File: src/JSBSim.cpp [hunk 3]
```diff
@@ -336,14 +336,14 @@ int real_main(int argc, char* argv[])
...
   if (!success) {
     PrintHelp();
     exit(-1);
   }
 
-  // *** SET UP JSBSIM *** //
+  // *** 设置仿真参数 *** //
   FDMExec = new JSBSim::FGFDMExec();
```
Comment (bcoconni) [inline src/JSBSim.cpp:347]: Same as above : leave the comments in English.

File: src/input_output/FGfdmSocket.cpp [hunk 4]
```diff
@@ -54,6 +54,9 @@ using std::cerr;
 using std::endl;
 using std::string;
 
+// link with Ws2_32.lib
+#pragma comment (lib, "Ws2_32.lib")
+
 namespace JSBSim {
```
Comment (bcoconni) [inline src/input_output/FGfdmSocket.cpp:60]: Do not include the libraries with `#pragma`. Add it to the MSVC project.

**Comment by lookingforfeng** [comment]
sorry ,this is the first time i try to give a pull requset , i just want give a solution to issue #254,not to send the change comments(its just  for my own use),i think you are right, i shoud not  include the libraries with #pragma.  and i will close the requset. sorry againe

**Comment by bcoconni** [comment]
> sorry ,this is the first time i try to give a pull requset , i just want give a solution to issue #254

No problem, this was perfectly clear. The comments @seanmcleod and myself made were not meant to reject your pull request but rather to make sure it meets JSBSim usual standards. Your contribution is appreciated even though we have requested some modifications to it.
> i think you are right, i shoud not include the libraries with #pragma. and i will close the requset. sorry againe

It is perfectly fine to submit a new PR (i.e. Pull Request) but this was not necessary. Actually, every commit you push to a branch submitted as a PR are cumulating into the PR in question. This is by the way the reason why the comments translation you made have ended up in the PR.

Basically, you can update your PR simply by pushing new commits to the branch you submitted. We will then review the updated proposal and once we are happy with it, it will be merged to JSBSim.

**Comment by lookingforfeng** [comment]
@bcoconni ok ,i got it ,thank you
